Article/Article.js

```diff
@@ -1,20 +1,34 @@
-// Because classes are not hoised you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
+// Because classes are not hoisted you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
 
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
+
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = domElement.querySelector('.expandButton');
+
     // Using your expandButton reference, update the text on your expandButton to say "expand"
+    this.expandButton.textContent = 'Expand';
     
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', this.expandArticle.bind(this));
   }
 
   expandArticle() {
     // Using our reference to the domElement, toggle a class to expand or hide the article.
+    //  this.domElement.classList.toggle('article-open')
+    if(this.domElement.classList.contains('article-open') === false ) {
+    
+      this.expandButton.textContent = 'Close';
+      TweenMax.to(this.domElement, 1, {className: '+=article-open'});
```

Comment:
You did a nice job on this, but I would suggest being mindful of the next developer when you are naming.